src/Equinor.ProCoSys.IPO.Command/InvitationCommands/DeleteAttachment/DeleteAttachmentCommandHandler.cs

```diff
@@ -42,6 +42,7 @@ public async Task<Result<Unit>> Handle(DeleteAttachmentCommand request, Cancella
             await _blobStorage.DeleteAsync(fullBlobPath, cancellationToken);
 
             invitation.RemoveAttachment(attachment);
+            _invitationRepository.RemoveAttachment(attachment);
```

Comment:
Add tests in DeleteAttachmentCommandHandlerTests, asserting that attachment table got 1 less attachment